enhancements/installer/alibaba-cloud-ipi.md

```diff
@@ -0,0 +1,195 @@
+---
+title: Alibaba Cloud IPI
+authors:
+  - "@lemondlut"
+reviewers:
+  - "@bradteng"
+  - "@rosawy"
+approvers:
+  - "@?"
+creation-date: 2021-03-25
+last-updated: 2021-03-25
+status: implementable
+
+---
+
+# Alibaba Cloud IPI
+
+## Release Signoff Checklist
+
+- [X] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This enhancement covers adding support to install OpenShift clusters to Alibaba Cloud and details for the Installer to create infrastructure and run an OpenShift cluster on Alibaba Cloud.
+
+## Motivation
+
+Currently, openshift4.x does not support Alibaba Cloud via IPI installation. However, the feedback team got is that there is a lot of demand for this kind of installation.
+
+Users interest in running Openshift in Alibaba Cloud can already do it manually. (Link TBD will be added after UPI document is published), but IPI installation will make it much simpler and automated.
+
+### Goals
+
+1. Installer creates infrastructure and installs OpenShift on Alibaba Cloud.
+2. Installer provides few but useful and comprehensive customizations through the install-config.yaml file.
+3. RHCOS images are built and published to Alibaba as part of the images pipeline process.
+4. Add a CI job that executes the provisioning steps to test the Alibaba IPI installer.
+
+### Non-Goals
+
+## Proposal
+
+### User Stories
+
+Enables `openshift-install` to create and destroy a cluster on Alibaba Cloud, which is managed and released and covered by CI tests.
+
+### Implementation Details/Notes/Constraints [optional]
```

Comment:
On some cloud platforms the user must perform some minimal steps to prepare the account to work with the installer (other than acquire a token). For example, in some clouds the user must upload their own RHCOS image, because there is not a shared image.

Are there any account setup requirements like this for Alibaba?

Reply:
- There might be some DNS resolution - private zone creation.
- RAM user - (role access management)
  - There might be additional agreements for permissions for the RAM user to call APIs
- AK secret credential

Please place this setup process into the enhancement.